index.html

```diff
@@ -10,14 +10,21 @@
 
     <div class="header">
         <div>
+          <div class="nav-cover">
             <nav>
                 <div class="logo"><a href="">ECHEZONA</a></div>
                 <a href="#" class="toggle-button">
                     <svg class="bar" width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
                     <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z" fill="#6070FF"/>
                     </svg>
                 </a>
+                <ul id="hamburger-menu">
+                  <li class="nav-links"><a href="#portfolio">Portfolio</a></li>
+                  <li class="nav-links"><a href="#about">About</a></li>
+                  <li class="nav-links"><a href="#contact">Contact</a></li>
+              </ul>
```

Comment:
You have done great work implementing the changes given by the previous reviewers😉 but there are still some changes suggested by the previous reviewers that haven't been implemented,  currently your intro section is looking different from the design, kindly make a comparison and try to fix them🤗

your Intro section
<img width="816" alt="your___DExktop" src="https://user-images.githubusercontent.com/64643995/213135054-73b17b8b-c2c6-46e6-be22-e7fb168fb171.png">

The design
![design_desktop](https://user-images.githubusercontent.com/64643995/213135396-63403769-6ec6-47ce-a517-d18bb7403681.png)

also, your images should occupy about 50% of the card width and 100% of the hight.  kindly compare the design with your project and make the necessary changes🤗